crowdin/messages.json

```diff
@@ -1088,6 +1083,7 @@
   "1524033858": "What is DTrader?",
   "1525885795": "As a payment agent, will I receive commissions from Deriv?",
   "1527682674": "Register for Deriv P2P by logging into your Deriv account in the app.",
+  "1528001864": "DMT5 - ${current_step}",
```

Comment:
doesn't look like a translation pattern - `${current_step}`
Did we add a new pattern?